### ISSUE TITLE
Add transpose functionality

### ISSUE THREAD

**Issue by ellliottt** [PyTables/PyTables#579]
With big arrays this will result in a MemoryError:

`transpose = f.root.matrix[:].T`

There should be an efficient built-in method for transposing arrays without loading into memory. This would be good because getting individual column slices is slow.

**Comment by harzus** [comment]
yep, that would be nice. otherwise i can't think of a better way than loading it into numpy and do it there
